src/commands/send_envelope.rs

```diff
@@ -26,6 +26,12 @@ pub fn make_command(command: Command) -> Command {
                 .required(true)
                 .help("The path or glob to the file(s) in envelope format to send as envelope(s)."),
         )
+        .arg(
+            Arg::new("raw")
+                .long("raw")
+                .action(ArgAction::SetTrue)
```

Comment:
```suggestion
```

Reply:
That does not appear to work. Note that `matches.get_flag("raw")` is used later to get the value, and AFAICT that requires `ArgAction::SetTrue` or `ArgAction::SetFalse`.

Reply:
That's what I changed it to :) I just prefer being consistent